### PR TITLE
Miguel/mat 3347 new chain spec for dev testnet

### PR DESCRIPTION
**Pull request by MiguelDD1**
Minor fixes

### CODE REVIEW

File: tests/app_id/src/app.ts [hunk 1]
```diff
@@ -59,64 +61,45 @@ async function main () {
   // Create the API and wait until ready
   const api = await createApi(); 
 
+  const keyring = new Keyring({ type: 'sr25519'});
+  const mnemonic = `inject fiscal misery fiscal success weasel black tube satisfy rural sauce reveal`;
```
Comment (prabal-banerjee) [inline tests/app_id/src/app.ts:65]: Let's avoid pushing mnemonic on repo.

**Review by prabal-banerjee — CHANGES_REQUESTED** [review]

File: tests/app_id/src/app.ts (hunk 1 above)
Comment (MiguelDD1) [reply tests/app_id/src/app.ts]: Fixed.

Comment (kroos47) [reply tests/app_id/src/app.ts]: @prabal-banerjee @MiguelDD1 would you like to add the new script here for getting back the data without any extrisic ID? or leave it for now ?

**Review by prabal-banerjee — APPROVED** [review]

**Review by aterentic-ethernal — APPROVED** [review]

**Review by kroos47 — APPROVED** [review]

File: tests/app_id/src/app.ts (hunk 1 above)
Comment (MiguelDD1) [reply tests/app_id/src/app.ts]: @kroos47 Let's add it in a new small PR.

**Review by prabal-banerjee — APPROVED** [review]

**Pull request merged**